### PR TITLE
change default value of profilePic

### PR DESCRIPTION
**Pull request by CHOCOLVT**

### CODE REVIEW

File: app/src/main/java/ch/epfl/sdp/blindwar/ui/ProfileActivity.kt [hunk 1]
```diff
@@ -41,15 +39,13 @@ class ProfileActivity : AppCompatActivity() {
...
             val eloView = findViewById<TextView>(R.id.eloDeclarationView)
             val profileImageView = findViewById<ImageView>(R.id.profileImageView)
-            if (user != null) {
-                nameView.text = user.firstName
-                emailView.text = user.email
-                eloView.text = user.userStatistics.elo.toString()
-
-                val imagePath = user.profilePicture.toString()
-                if (imagePath != null && imagePath != "null") {
+            user?.let{
```
Comment (Marty99-dev) [inline app/src/main/java/ch/epfl/sdp/blindwar/ui/ProfileActivity.kt:42]: We are becoming Kotlin Master guys !!!!

File: app/src/main/java/ch/epfl/sdp/blindwar/ui/ProfileActivity.kt [hunk 2]
```diff
@@ -41,15 +39,13 @@ class ProfileActivity : AppCompatActivity() {
...
+                nameView.text = it.firstName
+                emailView.text = it.email
+                eloView.text = it.userStatistics.elo.toString()
+                if (it.profilePicture != "") {
```
Comment (Marty99-dev) [inline app/src/main/java/ch/epfl/sdp/blindwar/ui/ProfileActivity.kt:46]: I think that using the isNotEmpty methods is cleaner

File: app/src/main/java/ch/epfl/sdp/blindwar/ui/UserNewInfoActivity.kt [hunk 3]
```diff
@@ -52,9 +52,9 @@ class UserNewInfoActivity : AppCompatActivity() {
                 lastName.setText(it.lastName)
                 pseudo.setText(it.pseudo)
                 if (!intent.getBooleanExtra("newUser", false)) {
-                    if (it.profilePicture != "null") {
+                    if (it.profilePicture != "") {
```
Comment (Marty99-dev) [inline app/src/main/java/ch/epfl/sdp/blindwar/ui/UserNewInfoActivity.kt:55]: That was definitely a must to do

**Review by Marty99-dev — APPROVED** [review]

**Review by ghemDD — APPROVED** [review]

File: app/src/main/java/ch/epfl/sdp/blindwar/ui/UserNewInfoActivity.kt [hunk 4]
```diff
@@ -52,9 +52,9 @@ class UserNewInfoActivity : AppCompatActivity() {
...
                         imageDatabase.dowloadProfilePicture(
-                            it.profilePicture!!,
+                            it.profilePicture,
```
Comment (marluxiaboss) [inline app/src/main/java/ch/epfl/sdp/blindwar/ui/UserNewInfoActivity.kt:57]: Indeed, we should never call this function if the profilePicture is Null.

**Review by marluxiaboss — APPROVED** [review]
This PR should fix the issue. I like that you change "null" to "" since it's what I've done before, we shouldn't use different default values across the program since it can create bugs. Also, there are a lot copy paste of code that I've already created. We should not have to copy paste. Trying to have profilePicture related thing on its own class or fragment is definitely something that we should work on.

**Pull request merged**